plugins/rikki/loungeviews/components/viewmatch/default.htm

```diff
@@ -124,7 +124,11 @@ <h3 class="card-header text-center" style="background-color: #fff">
         <div class="tab-content">
             {% for game in __SELF__.match.games %}
             <div class="tab-pane {% if loop.first %}active{% endif %}" id="game{{game.id}}" role="tabpanel">
-                {% component 'gameStatistic' game_id = game.id %}
+                {% if loop.first %}
+                    {% component 'gameStatistic' game_id = game.id lazy = 'false' %}
+                {% else %}
+                    {% component 'gameStatistic' game_id = game.id lazy= 'true' %}
```

Comment:
nit: lazy =

Reply:
Also you can do {% component 'gameStatistic' game_id = game.id lazy = !loop.first %}